cogs/funnies.py

```diff
@@ -53,6 +53,9 @@ async def on_message(self, message: discord.Message):
 
         if message.content == "dc":
             await message.channel.send("https://autocompressor.net/av1?s=wfinqwL9")
-        
+
+        if self.bot.mogi["status"] and "9/11" in message.content.lower():
```

Comment:
please also add a check to make sure the amount of players in the mogi is actually 11